### PR TITLE
Updates Compute.Json to improve Documentation around PerformMaintenan…

### PR DESCRIPTION
**Pull request by srvar**
…… and ConvertToManagedDisks APIs

Updates Compute.Json to improve Documentation around PerformMaintenan… and ConvertToManagedDisks APIs

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by hyonholee — APPROVED** [review]

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json [hunk 1]
```diff
@@ -3108,7 +3113,7 @@
           "VirtualMachines"
         ],
         "operationId": "VirtualMachines_PerformMaintenance",
-        "description": "The operation to perform maintenance on a virtual machine.",
+        "description": "Shuts down the virtual machine, moves it to a already updated node, and powers it back on during the self-service phase of planned maintenance.",
```
Comment (fengzhou-msft) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json:3116]: `an` already updated node

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json [hunk 2]
```diff
@@ -5552,7 +5557,7 @@
           "VirtualMachineScaleSetVMs"
         ],
         "operationId": "VirtualMachineScaleSetVMs_PerformMaintenance",
-        "description": "Performs maintenance on a virtual machine in a VM scale set.",
+        "description": "Shuts down the virtual machine in a VMScaleSet, moves it to a already updated node, and powers it back on during the self-service phase of planned maintenance.",
```
Comment (fengzhou-msft) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json:5560]: `an` already updated node

**Comment by grizzlytheodore** [comment]
LGTM, just description and example update

**Comment by madewithsmiles** [comment]
@srvar Can you please address @fengzhou-msft's comments?

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json [hunk 3]
```diff
@@ -5901,7 +5906,7 @@
           "VirtualMachineScaleSetVMs"
         ],
         "operationId": "VirtualMachineScaleSetVMs_PerformMaintenance",
-        "description": "Performs maintenance on a virtual machine in a VM scale set.",
+        "description": "Shuts down the virtual machine in a VMScaleSet, moves it to a already updated node, and powers it back on during the self-service phase of planned maintenance.",
```
Comment (fengzhou-msft) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json:5909]:
```suggestion
        "description": "Shuts down the virtual machine in a VMScaleSet, moves it to an already updated node, and powers it back on during the self-service phase of planned maintenance.",
```

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json (hunk 1 above)
Comment (fengzhou-msft) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json:3116]:
```suggestion
        "description": "Shuts down the virtual machine, moves it to an already updated node, and powers it back on during the self-service phase of planned maintenance.",
```

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json (hunk 2 above)
Comment (fengzhou-msft) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json:5560]:
```suggestion
        "description": "Shuts down the virtual machine in a VMScaleSet, moves it to an already updated node, and powers it back on during the self-service phase of planned maintenance.",
```

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json [hunk 4]
```diff
@@ -3405,7 +3410,7 @@
           "VirtualMachines"
         ],
         "operationId": "VirtualMachines_PerformMaintenance",
-        "description": "The operation to perform maintenance on a virtual machine.",
+        "description": "Shuts down the virtual machine, moves it to a already updated node, and powers it back on during the self-service phase of planned maintenance.",
```
Comment (fengzhou-msft) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json:3413]:
```suggestion
        "description": "Shuts down the virtual machine, moves it to an already updated node, and powers it back on during the self-service phase of planned maintenance.",
```

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**